src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -35,14 +36,15 @@ export default function HelperErrorText(
     props: React.PropsWithChildren<HelperErrorTextProps>
 ) {
     const {
+        attributes,
         id,
         blockName,
-        baseClass = "helper-text",
         isError,
         ariaLive = "polite",
         ariaAtomic = true,
     } = props;
 
+    let baseClass = "helper-text";
```

Comment:
Is `props.baseClass` used at all here, then?  If not, take it out of `HelperErrorTextProps`

Reply:
I'm not seeing it in `HelperErrorTextProps`... do you mean `HelperErrorTextOptions`?

Reply:
Ended up removing `HelperErrorTextOptions`—looks like they could be safely deprecated at this point.